molecule/fpm/Dockerfile.j2

```diff
@@ -5,7 +5,14 @@ FROM {{ item.registry.url }}/{{ item.image }}
 {% else %}
 FROM {{ item.image }}
 {% endif %}
+
 # install minimal packages for debian slim images
+{% if item.image == 'debian:bullseye-slim' %}
+RUN apt-get update && \
+    apt-get install -y python3 sudo bash ca-certificates iproute2 systemd systemd-sysv python3-pip && \
+    apt-get clean
+{% else %}
```

Comment:
I would change these conditions:
if OS-> Debian Stretch then install python2 packages
else
install python3 packages.
I think it's better with a view to the moment when we will drop Debian Stretch support.

Reply:
But buster still uses python2, rigth?

Reply:
Nope, you can use it, but you can survive with only python3 (see mysql_role PR #80).

Reply:
In the end, we will keep it this way in this role. It is true that we could survive only with python3, but Debian buster uses python2 by default.